docs/tutorials/openfoam/spack-openfoam.yaml

```diff
@@ -153,25 +158,37 @@ deployment_groups:
           mpirun -n 60 -npernode 30 -hostfile hostfile snappyHexMesh -overwrite -parallel
           mpirun -n 60 -npernode 30 -hostfile hostfile potentialFoam -parallel
           mpirun -n 60 -npernode 30 -hostfile hostfile simpleFoam -parallel
+
+  - id: compute_nodeset
+    source: community/modules/compute/schedmd-slurm-gcp-v6-nodeset
+    use: [network1]
+    settings:
+      name: ns
```

Comment:
I believe name is no longer required, I would remove it if not required.

Reply:
Removed